src/plugins/data_source/README.md

```diff
@@ -1,9 +1,73 @@
-# data_source
+# DataSource Plugin
 
 An OpenSearch Dashboards plugin
 
-This plugin introduces OpenSearch data source into OpenSearch Dashboards, and provides related functions to connect to OpenSearch data sources.
+This plugin introduces multiple OpenSearch data sources into OpenSearch Dashboards, and provides related functions to connect to OpenSearch data sources.
 
+## Configuration
+Update the following flags in the opensearch_dashboards.yml file to apply changes.
+
+1. The dataSource plugin is by default disabled, to enable it:
+`data_source.enabled: true`
+
+2. The audit trail is by default enabled for logging the accessing to data source, to disable it:
```

Comment:
we can add a link to our plugin config schema definition at https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data_source/config.ts here